### PR TITLE
CDSR-2020: Claims in progress

### PR DESCRIPTION
**Pull request by airudah2000**
Implements search component for claims in progress.

Fixes UI issue with search component

Other small fixes

### CODE REVIEW

File: app/views/claims_closed.scala.html [hunk 1]
```diff
@@ -29,11 +29,9 @@
 @(model: ClosedClaimListViewModel,
   caseStatusHints: DropdownHints,
   searchForm: Form[String],
-  postAction: Call,
-  companyName: String,
-  eori: String)(implicit request: IdentifierRequest[_], messages: Messages, appConfig: AppConfig)
+  postAction: Call)(implicit request: IdentifierRequest[_], messages: Messages, appConfig: AppConfig)
 
-@key = @{"claim-list-closed"}
+@key = @{"claim.list.closed"}
```
Comment (haseemtasheel) [inline app/views/claims_closed.scala.html:34]: I think we have kept `-` everywhere else

Comment (airudah2000) [reply app/views/claims_closed.scala.html:34]: No we haven't @haseemtasheel. This repo uses dots everywhere so we should stay consistent with that pattern. If we need to change it to hyphens like in CDSR frontend, then we should create a separate (tech-debt) ticket for it.

File: app/views/components/search.scala.html [hunk 2]
```diff
@@ -17,28 +17,25 @@
...
 
-<br/>
-@subHeading(Html(messages(s"$key.title")), classes = "govuk-heading-m")
-@paragraph(Html(messages(s"$key.desc")))
+    <br/>
+    @subHeading(Html(messages(s"$key.title")), classes = "govuk-heading-m")
+    @paragraph(Html(messages(s"$key.desc")))
 
-@formWithCSRF(postAction, 'novalidate -> "novalidate", 'style -> "display: flex; gap: 5px; flex-direction: row;", 'class -> "govuk-!-width-two-thirds") {
-    @input(
```
Comment (airudah2000) [inline app/views/components/search.scala.html:36]: The problem with this @input object is that it creates a wrapper div with class "govuk-form-group" which wraps the input only, whereas we need that div to wrap both the input and submit button. 
This is why we were having difficulty showing the button on the same line.

File: app/views/claims_closed.scala.html (hunk 1 above)
Comment (mghmay) [reply app/views/claims_closed.scala.html:34]: We have changed the style to reflect the style used in CDS-reimbursement, so the dashes are in the key, we then use the key on the view

File: app/views/components/search.scala.html (hunk 2 above)
Comment (mghmay) [reply app/views/components/search.scala.html:36]: We have created a ticket for this for Tony

**Review by mghmay — APPROVED** [review]

**Pull request merged**